docs/reference/method/MongoDBModelIndexInfo-isBackground.txt

```diff
@@ -0,0 +1,53 @@
+=========================================
+MongoDB\\Model\\IndexInfo::isBackground()
```

Comment:
I'd propose deleting this method. I'm not sure it's relevant to users once the index has been created, and it's trivial to determine this via `offsetGet()`.